gensim/callbacks.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Abstract base class to build callbacks. Callbacks are used to apply custom functions over the model
```

Comment:
It may not make sense to make this inherit `ABC` and use `@abstractmethod` since not all the methods (`on_epoch_end`, `on_train_begin` etc.) need to be re-implemented -- just the one required by the user. Changing 'Abstract base class' to 'Base class` in docstring.